lib/tasks/taxonomy/analysis/taxons_tagged_to_content.rake

```diff
@@ -24,22 +24,33 @@ namespace :taxonomy do
         page += 1
       end
 
-      puts %w(organisations content_id base_path number_of_taxons).to_csv
+      content_items.each do |item|
```

Comment:
@alecgibson Thanks Alec, I don't know many of the methods for arrays or hashes. A lot of them seem to be interchangeable :/ Yeah a lot of reading will have to be done